Formula/allegro.rb

```diff
@@ -15,24 +15,18 @@ class Allegro < Formula
     sha256 "65a04aa3c0901264e54ca91f8982da085ea90bccabcb885fac054ba5219e19bd" => :mavericks
   end
 
-  devel do
-    url "http://download.gna.org/allegro/allegro-unstable/5.1.13.1/allegro-5.1.13.1.tar.gz"
-    sha256 "c6e6265c3d661d46270971b7fbf7db34c1873af62882a9ea4025ca1298edf14d"
-
-    depends_on "theora" => :recommended
-  end
-
   head do
-    url "https://github.com/liballeg/allegro5.git", :branch => "5.1"
-
-    depends_on "theora" => :recommended
+    url "https://github.com/liballeg/allegro5.git", :branch => "5.2"
   end
 
   depends_on "cmake" => :build
   depends_on "libvorbis" => :recommended
   depends_on "freetype" => :recommended
   depends_on "flac" => :recommended
   depends_on "physfs" => :recommended
+  depends_on "dumb" => :recommended
+  depends_on "libogg" => :recommended
+  depends_on "theora" => :recommended
```

Comment:
How come `dumb` and `libogg` were not required in the now removed development version, but are now needed in the new stable version that follows it?